infra/src/webapp.tf

```diff
@@ -64,6 +77,34 @@ locals {
 
     INTERNAL_STORAGE_CONNECTION_STRING = module.storage_account.primary_connection_string
 
+    # JIRA integration for Service review workflow
+    JIRA_NAMESPACE_URL                  = "https://pagopa.atlassian.net"
+    JIRA_PROJECT_NAME                   = "IES"
+    JIRA_TOKEN                          = data.azurerm_key_vault_secret.jira_token.value
+    JIRA_USERNAME                       = "github-bot@pagopa.it"
+    JIRA_CONTRACT_CUSTOM_FIELD          = ""
+    JIRA_DELEGATE_EMAIL_CUSTOM_FIELD    = "customfield_10084"
+    JIRA_DELEGATE_NAME_CUSTOM_FIELD     = "customfield_10087"
+    JIRA_ORGANIZATION_CF_CUSTOM_FIELD   = ""
+    JIRA_ORGANIZATION_NAME_CUSTOM_FIELD = "customfield_10088"
+
+    # Apim connection
+    AZURE_APIM                = "io-p-apim-api"
```

Comment:
Maybe we want to read these values from variables? It's a bit more verbose, but it would allow us to configure different environments if needed. 

(I mean: we can put the same values in `env/prod/terraform.tfvars`)

Reply:
[36a28ec](https://github.com/pagopa/io-services-cms/pull/33/commits/36a28ecff750a6ad77f16dd059186a37a72955cd)